diff -urN openrc-0.6.8/conf.d/network.Linux openrc-0.6.8_modified//conf.d/network.Linux
--- openrc-0.6.8/conf.d/network.Linux	2010-12-07 22:32:44.000000000 -0600
+++ openrc-0.6.8_modified//conf.d/network.Linux	2011-01-11 12:18:01.000000000 -0600
@@ -12,6 +12,9 @@
 #defaultiproute="via 192.168.0.1"
 #defaultiproute6="via 2001:a:b:c"
 
+# You can add additional routes.  Entries will be append to "ip route add".
+#iproutes=('to 192.168.12/24 via 192.168.0.1' 'to 2001:49e::/48 via 2001:49e::1')
+
 # ip doesn't handle MTU like ifconfig, but we can do it like so
 #ifup_eth0="ip link set \$int mtu 1500"
 
diff -urN openrc-0.6.8/init.d/network.in openrc-0.6.8_modified//init.d/network.in
--- openrc-0.6.8/init.d/network.in	2010-12-07 22:32:44.000000000 -0600
+++ openrc-0.6.8_modified//init.d/network.in	2011-01-11 12:11:25.000000000 -0600
@@ -313,6 +313,27 @@
 		eend $?
 	fi
 
+	if [ -n "$routes" ]; then
+		count=${#routes[@]}
+		i=0
+		while [ "$i" -lt "$count" ]; do
+			ebegin "Setting route ${routes[$i]}"
+			route add ${routes[$i]}
+			eend $?
+			((i++))
+		done;
+	elif [ -n "$iproutes" ]; then
+		count=${#iproutes[@]}
+		i=0
+		while [ "$i" -lt "$count" ]; do
+			ebegin "Setting route ${iproutes[$i]}"
+			ip route add ${iproutes[$i]}
+			eend $?
+			((i++))
+		done;
+	fi
+
+
 	return 0
 }
 
